mlflow/pytorch/__init__.py

```diff
@@ -285,6 +285,106 @@ def gen_data():
     )
 
 
+def log_state_dict(state_dict, artifact_path, pickle_module=None, **kwargs):
+    """
+    Log a PyTorch model as an MLflow artifact for the current run.
+
+    :param state_dict: PyTorch model state dict to be saved.
```

Comment:
[Question about TorchServe] I think there are two types of state dicts.

1. state_dict that represents a model:

```python
model.state_dict()
```

2. state_dict that represents a checkpoint (or might something else):

```python
{
   "model": model.state_dict(),
   "optim": optimiser.state_dict(),
   ...
}
```

Can we serve the checkpoint state dict with TorchServe?

Reply:
We need to save it as `model.state_dict()`. confirmed it with @gchauhang .